graphql/graphql.go

```diff
@@ -236,6 +236,14 @@ func (t *Transaction) GasPrice(ctx context.Context) (hexutil.Big, error) {
 			}
 		}
 		return hexutil.Big(*tx.GasPrice()), nil
+	case types.CeloDenominatedTxType:
+		if t.block != nil {
+			if baseFee, _ := t.block.BaseFeePerGasForCurrency(ctx, nil); baseFee != nil {
+				// price = min(tip, gasFeeCap - baseFee) + baseFee
+				return (hexutil.Big)(*math.BigMin(new(big.Int).Add(tx.GasTipCap(), baseFee.ToInt()), tx.GasFeeCap())), nil
+			}
+		}
+		return hexutil.Big(*tx.GasPrice()), nil
```

Comment:
Can't you keep the same code for all types if you use `tx.DenominatedFeeCurrency` instead of `tx.FeeCurrency`

Reply:
Yes, I wrote this before adding DenominatedFeeCurrency and I missed to go back and change it in the previous changes. Thanks